osquery/tables/system/hash.cpp

```diff
@@ -182,6 +186,21 @@ bool FileHashCache::load(const std::string& path, MultiHashes& out) {
   return true;
 }
 
+std::string genSsdeepForFile(const std::string& path) {
+#ifndef WINDOWS
```

Comment:
ssdeep looks like the feature every OS could benefit. Is it possible to implement it for windows also?

Reply:
I agree that Windows can benefit from it.
But since osquery dependency management is completely different for Windows AFAIK, I suggest that this is done in a separate PR.